### PR TITLE
new test to validate that contains_entity works

### PR DESCRIPTION
**Pull request by jwillemsen**
issue #2048

### CODE REVIEW

**Comment by jwillemsen** [comment]
Test extension which now succeeds because #2068 has been merged

**Review by mitza-oci — CHANGES_REQUESTED** [review]
looks like there are supposed to be more files in this PR (source code for test)

**Comment by jwillemsen** [comment]
Merged with master, test now passes with the dp changes

**Comment by jwillemsen** [comment]
> looks like there are supposed to be more files in this PR (source code for test)

Added

File: bin/dcps_short_tests.lst [hunk 1]
```diff
@@ -54,6 +54,13 @@ tests/DCPS/DpShutdown/run_test.pl tcp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO
 tests/DCPS/DpShutdown/run_test.pl shmem: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_SHMEM
 tests/DCPS/DpShutdown/run_test.pl udp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
 tests/DCPS/DpShutdown/run_test.pl multicast: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_MCAST
+
+tests/DCPS/ContainsEntity/run_test.pl: RTPS
+tests/DCPS/ContainsEntity/run_test.pl tcp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
+tests/DCPS/ContainsEntity/run_test.pl shmem: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_SHMEM
+tests/DCPS/ContainsEntity/run_test.pl udp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
+tests/DCPS/ContainsEntity/run_test.pl multicast: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_MCAST
```
Comment (mitza-oci) [inline bin/dcps_short_tests.lst:62]: Does the test need to run 5 times with different transports?  It's testing behavior at the DCPS layer, seems like the transport type shouldn't matter.

File: tests/DCPS/ContainsEntity/ContainsEntity.mpc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+project(DDS*) : dcpsexe, dcps_test, dcps_cm, dcps_transports_for_test {
+  exename = containsentity
+
+  Idl_Files {
+  }
+
```
Comment (mitza-oci) [inline tests/DCPS/ContainsEntity/ContainsEntity.mpc:6]:
```suggestion
```

**Review by mitza-oci — CHANGES_REQUESTED** [review]

File: bin/dcps_short_tests.lst (hunk 1 above)
Comment (jwillemsen) [reply bin/dcps_short_tests.lst]: Just running it with RTPS only as part of the test suite

**Review by mitza-oci — APPROVED** [review]

**Pull request merged**